### PR TITLE
OTT-1738 :: provide exclusion config to adpod through request ext

### PR DESCRIPTION
**Pull request by pm-nikhil-vaidya**
Sample exclusion config:
```
{
  "ext": {
    "adpod": {
      "exclusion": {
        "iabcategory": [
          "pod1",
          "pod2"
        ],
        "advertiserdomain": [
          "pod1"
        ]
      }
    }
  }
} 
```

### CODE REVIEW

File: endpoints/openrtb2/ctv/adpod/dynamic_adpod.go [hunk 1]
```diff
@@ -55,6 +57,15 @@ func NewDynamicAdpod(pubId string, imp openrtb2.Imp, adpodExt openrtb_ext.ExtVid
...
 		}
+
+		if exclusion.AdvertiserDomainExclusion {
+			adpodExt.AdPod.AdvertiserExclusionPercent = ptrutil.ToPtr(0)
+		}
+
+		if exclusion.IABCategoryExclusion {
+			adpodExt.AdPod.IABCategoryExclusionPercent = ptrutil.ToPtr(0)
+		}
+
```
Comment (ShriprasadM) [inline endpoints/openrtb2/ctv/adpod/dynamic_adpod.go:68]: @pm-nikhil-vaidya : shouldn't it be `100`?

**Review by ShriprasadM — COMMENTED** [review]
@pm-nikhil-vaidya  : LGTM.  Just one comment left.

File: endpoints/openrtb2/ctv/adpod/dynamic_adpod.go (hunk 1 above)
Comment (pm-nikhil-vaidya) [reply endpoints/openrtb2/ctv/adpod/dynamic_adpod.go:68]: Here we are reusing the old configurations, where 0 means exclusion and 100 means no exclusion, its a invert condition

**Review by ShriprasadM — DISMISSED** [review]
LGTM

**Review by Pubmatic-Dhruv-Sonone — APPROVED** [review]

**Review by pm-viral-vala — APPROVED** [review]

**Review by Ankit-Pinge — APPROVED** [review]

**Pull request merged**